libr/core/cmd_print.c

```diff
@@ -5835,6 +5836,7 @@ static int cmd_print(void *data, const char *input) {
 				r_cons_newline ();
 			}
 		}
+		r_config_set_i (core->config, "asm.segoff", segoff);
```

Comment:
```suggestion
		r_config_set_b (core->config, "asm.segoff", segoff);
```

Reply:
unfortunately I need to revert that part because it is more complicated problem than I though at first place. #19599

Reply:
Ok